docs/pages/components/side-navigation.md

```diff
@@ -358,13 +365,13 @@ The lists in both sections (Main and Utility) should have the `fd-nested-list--c
                 </a>
             </li>
         </ul>
-    </div>
-    <div class="fd-side-nav__utility">
-        <ul class="fd-nested-list fd-nested-list--compact">
+    </nav>
+    <nav class="fd-side-nav__utility">
+        <ul class="fd-nested-list fd-nested-list--compact" aria-label="Utility Menu">
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--compare"></span>
-                    <span class="fd-nested-list__title">Level 1 Item</span>
+                    <span aria-hidden="true" class="fd-nested-list__icon sap-icon--compare"></span>
+                    <a class="fd-nested-list__title" href="#">Level 1 Item</a>
```

Comment:
Should this remain as a span, as it's not a child item, similar to the other list items? And it may be causing a rendering issue:

![Capture](https://user-images.githubusercontent.com/22511993/82618230-52c8c100-9b87-11ea-9d73-8f9fc04b6fbc.PNG)